src/libraries/System.Formats.Tar/System.Formats.Tar.sln

```diff
@@ -1,4 +1,8 @@
-﻿Microsoft Visual Studio Solution File, Format Version 12.00
+﻿
```

Comment:
Please re-generate this project file via slngen by invoking the following command:

`dotnet.cmd build src\libraries\slngen.proj /p:SolutionNameOverride=System.Formats.Tar`

We use that tool to (re-)generate our solution files and it avoids these headers that pin the file to a minimum VS version.

Reply:
Sure, no problem.

Reply:
@ViktorHofer I got this error when trying with both dotnet 7.0-preview7 and 7.0-rc1. Am I supposed to use 6.0 for this?

```
MSBuild version 17.4.0-preview-22428-01+14c24b2d3 for .NET
  Determining projects to restore...
  Restored C:\Users\calope\source\repos\runtime\src\libraries\slngen.proj (in 318 ms).
  C:\Program Files\dotnet
  The .NET SDK 7.0.100-rc.1.22431.12 is not supported
C:\Users\calope\source\repos\runtime\src\libraries\slngen.proj(24,5): error MSB3073: The command ""C:\Users\calope\source\repos\runtime\dotnet.cmd" slngen -p SlnGenMainProject=System.Formats.Tar --launch false 
 --nologo "C:\Users\calope\source\repos\runtime\src\libraries\System.Formats.Tar\System.Formats.Tar.proj"" exited with code -1.

Build FAILED.

C:\Users\calope\source\repos\runtime\src\libraries\slngen.proj(24,5): error MSB3073: The command ""C:\Users\calope\source\repos\runtime\dotnet.cmd" slngen -p SlnGenMainProject=System.Formats.Tar --launch false
 --nologo "C:\Users\calope\source\repos\runtime\src\libraries\System.Formats.Tar\System.Formats.Tar.proj"" exited with code -1.
    0 Warning(s)
    1 Error(s)

Time Elapsed 00:00:09.70
```

Reply:
It failed and didn't modify the sln at all, but it did generate a new proj file, which was placed under `runtime\src\libraries\System.Formats.Tar\System.Formats.Tar.proj` and the contents were these:

```xml
<!-- This file is used by slns.proj. -->
<Project Sdk="Microsoft.Build.Traversal">
  <PropertyGroup>
    <IncludeInSolutionFile>false</IncludeInSolutionFile>
  </PropertyGroup>
  <ItemGroup>
    <ProjectReference Include="$(MSBuildThisFileDirectory)ref\*.*proj" />
    <ProjectReference Include="$(MSBuildThisFileDirectory)src\*.*proj" />
    <!-- Keep in sync with https://github.com/dotnet/runtime/blob/4d82932722b56924598708f991961a75e5a90b36/src/libraries/Directory.Build.props#L31. -->
    <ProjectReference Include="$(MSBuildThisFileDirectory)tests\**\*.Tests.*proj" />
    <ProjectReference Include="$(MSBuildThisFileDirectory)tests\**\*.UnitTests.*proj" />
  </ItemGroup>
</Project>

```

Reply:
Yeah the auto-generated file is deleted later but as something failed, it remained.

> The .NET SDK 7.0.100-rc.1.22431.12 is not supported

This seems wrong. Trying to repro local.

Reply:
Found the issue. Will submit a fix and update all solution files all-together.

Reply:
Ok so I guess I can close this PR, since I wanted to also revert the csproj change.